### ISSUE TITLE
Installs unstripped shared libraries

### ISSUE THREAD

**Issue by yurivict** [sjaehn/BSEQuencer#8]
```
Warning: 'lib/lv2/BSEQuencer.lv2/BSEQuencer.so' is not stripped consider trying INSTALL_TARGET=install-strip or using ${STRIP_CMD}
Warning: 'lib/lv2/BSEQuencer.lv2/BSEQuencer_GUI.so' is not stripped consider trying INSTALL_TARGET=install-strip or using ${STRIP_CMD}
```

**Comment by sjaehn** [comment]
Fixed in 6054e82254ffcc1a1e8035c43c87893a35708b0f using $(INSTALL) -s

**Comment by sjaehn** [comment]
Finally, rule "install-strip" added in 1b03eec868b4bad30c7141e4bcb4b13153a3c8d6 . Thus, use `sudo make install-strip PREFIX=/usr` for stripped installation.
